docs/guide.rst

```diff
@@ -4,30 +4,30 @@ The jupyter_rfb guide
 Installation
 ------------
 
-To install use pip:
+Install with pip:
 
 .. code-block::
 
     pip install -U jupyter_rfb
 
-Or to install into an existing conda environment:
+Or to install into a conda environment:
 
 .. code-block::
 
     conda install -c conda-forge jupyter-rfb
 
-Developers, see the `readme <https://github.com/vispy/jupyter_rfb>`_ for a dev installation.
+If you plan to hack on this library, see the `readme <https://github.com/vispy/jupyter_rfb>`_ for a dev installation.
 
 
 Subclassing the widget
 ----------------------
 
-The provided :class:`RemoteFrameBuffer <jupyter_rfb.RemoteFrameBuffer>` class cannot do much by itself, but it serves as
+The provided :class:`RemoteFrameBuffer <jupyter_rfb.RemoteFrameBuffer>` class cannot do much by itself, but it provides
 a basis for widgets that want to generate images at the server, and be informed
 of user events. The way to use ``jupyter_rfb`` is therefore to create a subclass
-and implement two methods.
+and implement two specific methods.
 
-The first method to implement is ``get_frame()``, which should return an (uint8) numpy array:
+The first method to implement is ``.get_frame()``, which should return an (uint8) numpy array. For example:
```

Comment:
```suggestion
The first method to implement is ``.get_frame()``, which should return a uint8 numpy array. For example:
```

I'm not sure the parentheses do anything useful here and I could be wrong about the `an` to `a` but I don't think so (if you say "you-int" then it is a Y sound which is not a vowel sound so it should be `a`).